keps/sig-storage/20181111-extend-datasource-field.md

```diff
@@ -134,6 +155,8 @@ Given that the only implementation changes in Kuberenetes is to enable the featu
 ## Implementation History
 
 1.15 - Alpha status
+1.16 - Beta status
+1.18 - GA Status
```

Comment:
I would either remove the 1.18 - GA since it's not official yet, or add a note that we're targeting GA in 1.18